lib/index.js

```diff
@@ -19,10 +20,15 @@ class ServerlessAWSPseudoParameters {
 
     this.serverless.cli.consoleLog(`${chalk.yellow.underline('AWS Pseudo Parameters')}`);
 
+    var skipRegionReplace = this.skipRegionReplace;
+    if (skipRegionReplace) {
+      console.info('Skipping automatic replacement of regions with account region!');
```

Comment:
Can we make sure this uses the proper logging structure? `this.serverless.cli.consoleLog` - might as well update L85 as well while we're in here.